libjupedsim/include/jupedsim/jupedsim.h

```diff
@@ -482,21 +432,235 @@ JUPEDSIM_API void JPS_JourneyDescription_Free(JPS_JourneyDescription handle);
 /**
  * Opaque type of Generalized Centrifugal Force model state
  */
-typedef struct JPS_GeneralizedCentrifugalForceModelState_t const*
+typedef struct JPS_GeneralizedCentrifugalForceModelState_t*
     JPS_GeneralizedCentrifugalForceModelState;
 
+/**
+ * Read speed of this agent.
+ * @handle of the Agent to access.
```

Comment:
Isn't it `@param param_name` in doxygen?

Reply:
yes, on it